### ISSUE TITLE
`no-navigation-without-resolve`: Track array and object elements

### ISSUE THREAD

**Issue by marekdedic** [sveltejs/eslint-plugin-svelte#1485]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.

### What version of ESLint are you using?

master

### What version of `eslint-plugin-svelte` are you using?

master

### What did you do?

```svelte
<script>
  const navItems = [
    { name: 'Home', url: resolve('/') },
    { name: 'About', url: resolve('/about') }, 
  ]
</script>

{#each navItems as item}
  <a href={item.url}>{item.name}</a>
{/each}
```

```svelte
<script>
  const paths = [ resolve("/"), resolve("/blog") ];
</script>

{#each paths as path}
  <a href={path}>Click me</a>
{/each}
```

```svelte
<script>
    const testUrlsList = ["https://google.com", "https://bing.com"];
</script>

{each testUrlsList as url}
    <a href={url}>A random phrase</a>
{/each}
```

### What did you expect to happen?

No error

### What actually happened?

Error

### Link to **GitHub Repo** with Minimal Reproducible Example

Will make a PR.
